doc/man/8/cephfs-shell.rst

```diff
@@ -164,6 +164,23 @@ Usage :
 
 * file - name of the file
 
+ln
+--
+
+Create a hard link or a symbolic link to an existing file or directory.
+
+Usage:
+
+    ln [options] <target> [link_name]
+
+* target - file/directory of which link is to be created
```

Comment:
` * target - file/directory to which a link is to be created`

Reply:
Okay, will fix this.

Reply:
Done, thanks!